src/pages/storage/StorageVolumes.tsx

```diff
@@ -320,31 +312,29 @@ const StorageVolumes: FC = () => {
             ]),
         {
           className: "actions u-align--right",
-          content:
-            volume.type === "custom" ? (
-              <CustomStorageVolumeActions
-                volume={volume}
-                project={project}
-                snapshotDisabled={snapshotsDisabled}
-                className={classnames(
-                  "storage-volume-actions",
-                  "u-no-margin--bottom",
-                )}
-              />
-            ) : (
-              <StorageVolumeNameLink
-                volume={volume}
-                project={project}
-                isExternalLink
-                overrideName={`go to ${
-                  volume.type === "image" ? "images list" : "instance"
-                }`}
-                className={classnames(
-                  "storage-volume-actions",
-                  "u-align--right",
-                )}
-              />
-            ),
+          content: hasDetailPage(volume) ? (
+            <CustomStorageVolumeActions
+              volume={volume}
+              projectName={project}
+              className={classnames(
+                "storage-volume-actions",
+                "u-no-margin--bottom",
+              )}
+            />
+          ) : (
+            <StorageVolumeNameLink
+              volume={volume}
+              project={project}
+              overrideName={`go to ${
+                volume.type === "image"
+                  ? "images list"
+                  : volume.content_type === "iso"
+                    ? "custom ISOs"
+                    : "instance"
+              }`}
+              className={classnames("storage-volume-actions", "u-align--right")}
```

Comment:
this is a legacy from before, but I think we don't need to use `classnames()` in this file.